autospec/buildreq.py

```diff
@@ -901,6 +906,27 @@ def scan_for_configure(dirn):
     else:
         config.autoreconf = False
 
+    if buildpattern.default_pattern == "distutils3":
+        # Try and grab the pypi details for the package
+        pypi_api_url = f"https://pypi.python.org/pypi/{tname}/{tversion}/json"
+        pypi_json = download.do_curl(pypi_api_url)
+        if pypi_json:
+            package_pypi = json.loads(pypi_json.getvalue().decode("utf-8"))
```

Comment:
Should we catch `JSONDecodeError` in case pypi gave us garbage?

Reply:
Seems like a safe thing to do.